daemon/mgr/spec_linux.go

```diff
@@ -39,18 +39,21 @@ func populatePlatform(ctx context.Context, c *Container, specWrapper *SpecWrappe
 	}
 
 	// same with containerd use. or make it a variable
-	cgroupsParent := "default"
+	// set default cgroup parent
+	cgroupsParent := "/default"
+	if specWrapper.useSystemd {
+		cgroupsParent = "system.slice"
+	}
+
 	if c.HostConfig.CgroupParent != "" {
-		cgroupsParent = c.HostConfig.CgroupParent
+		cgroupsParent = filepath.Clean(c.HostConfig.CgroupParent)
 	}
 
-	// cgroupsPath must be absolute path
-	// call filepath.Clean is to avoid bad
-	// path just like../../../.../../BadPath
-	if !filepath.IsAbs(cgroupsParent) {
-		cgroupsParent = filepath.Clean("/" + cgroupsParent)
+	if specWrapper.useSystemd {
+		s.Linux.CgroupsPath = cgroupsParent + ":" + "pouch" + ":" + c.ID
```

Comment:
when use string connection, I recommend using `fmt.Sprintf("%s:pouch:%s", cgroupsParent, c.ID)`. Since using `+` could introduce some potential issues.